package.json

```diff
@@ -119,7 +119,7 @@
         }
     },
     "engines": {
-        "node": ">=14.0.0 <15.0.0",
+        "node": ">=18.16.0",
```

Comment:
I think `18.x` or `18.0.0` is safest for customers. those already on v18 won't need to upgrade again and is compatible with frontend-mono if teams plan to use a version between `18.14.2` and `18.16.0`